doc/Language/js-nutshell.pod6

```diff
@@ -0,0 +1,185 @@
+=begin pod :tag<convert>
+
+=TITLE Perl 6 from Node.js - Nutshell
+
+=SUBTITLE Learning Perl 6 from Node.js, in a nutshell.
+
+This page attempts to provide a way for users experienced in Node,js to learn
+Perl 6. Features shared between the two languages will be explained here, as
+well as major differences in syntax and features.
+
+This is not a tutorial for learning Perl 6; this is a reference for users who
+are already at an intermediate to advanced skill level with Node.js.
+
+=head1 Basic Syntax
+
+=head2 "Hello, world!"
+
+Let's start with the typical first program when learning new languages. In
+Node.js, a hello world program would be written like this:
+
+=begin code :lang<javascript>
+console.log('Hello, world!');
+=end code
+
+In Perl 6, there are two ways to write this in the same way:
+
+=begin code
+put 'Hello, world!';
```

Comment:
> since this is the most commonly used (from what I gather in #perl6, that is)

Whether you use one or another [depends on your intent](https://docs.perl6.org/language/faq#How_and_why_do_say,_put_and_print_differ?)